### PR TITLE
Add v2v function cases

### PR DESCRIPTION
**Pull request by chloerh**
- convert disk image using -if option with the wrong format

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

File: v2v/tests/cfg/v2v_options.cfg [hunk 1]
```diff
@@ -396,3 +396,16 @@
                             not_in_man = --debug-gc
                         - sles_opensuse:
                             not_in_man = 'Virt-v2v is able to import Xen guests from RHEL 5 Xen or SLES and openSUSE'
+                - if_wrong_format:
+                    only output_mode.none
```
Comment (chunfuwen) [inline v2v/tests/cfg/v2v_options.cfg]: is this filter conflict with line 403 ?

File: v2v/tests/cfg/v2v_options.cfg [hunk 2]
```diff
@@ -396,3 +396,16 @@
                             not_in_man = --debug-gc
                         - sles_opensuse:
                             not_in_man = 'Virt-v2v is able to import Xen guests from RHEL 5 Xen or SLES and openSUSE'
+                - if_wrong_format:
```
Comment (chunfuwen) [inline v2v/tests/cfg/v2v_options.cfg]: if_wrong_format to wrong_format_if ?

File: v2v/tests/cfg/v2v_options.cfg (hunk 1 above)
Comment (chloerh) [reply v2v/tests/cfg/v2v_options.cfg]: This line is about output_mode, L403 is about input_mode, therefore no conflict

File: v2v/tests/cfg/v2v_options.cfg (hunk 2 above)
Comment (chloerh) [reply v2v/tests/cfg/v2v_options.cfg]: 'if' is an option for v2v, I like my way better :)

**Pull request merged**